deployment/docker-compose/docker-compose.dev.yml

```diff
@@ -11,6 +11,8 @@ services:
       - .env
     volumes:
       - db_volume:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
```

Comment:
Let's just keep this open so we can connect do DB and debug!

Reply:
Sounds good